aten/src/ATen/native/EmbeddingBag.cpp

```diff
@@ -177,18 +177,19 @@ void fbgemm_spmdm_report_error_(
       index_t idx = indices[i];
       TORCH_CHECK(
           0 <= idx && idx < N,
-          "Index ",
-          i,
-          " is out of bounds: ",
+          "Index value ",
```

Comment:
Another possible phrasing would be "The {idx}-th index value is {i} which is not in the valid range[0, {N}[" which I feel would be clearer no?

Reply:
Yes, certainly -- I committed a new version which should capture the goal of your comments (note -- I believe you might have idx and i switched in your suggestion).

Thank you so much for your review, please let me know if this requires further iteration.